validators/currency.py

```diff
@@ -0,0 +1,216 @@
+from .utils import validator
+
+
+@validator
+def iso_4217(value, case_sensitive=True):
```

Comment:
How about we'd just call this validator ``currency_code`` instead of ``iso_4217``. We could keep the comment block as it is describing that the currency code validator is based on ISO 4217